MobileWallet/Common/UserFeedback/UserFeedback.swift

```diff
@@ -62,7 +62,7 @@ class UserFeedback {
         let errorFeedbackView = FeedbackView()
 
         var descriptionText = description
-        if TariSettings.shared.environment == .debug, let e = error {
+        if TariSettings.shared.environment == .debug, let e = error, description != e.localizedDescription {
```

Comment:
This line is probably not needed. The `localizedDescription ` is only appended to the error description in debug. Run the app in release mode and you'll see the description will show as you expect

Reply:
no, if we do not change this line we can see duplicate some of lines in debug